### ISSUE TITLE
can't change auto export frequency and export suggestion

### ISSUE THREAD

**Issue by Sree-Siddhaarth** [cs0x7f/cstimer#368]
Currently csTimer has the autoexport frequency set to 100. It would be better if it is changable. I prefer to have it export every 20 or 25 solves.

One more thing. Having able to specify the folder where the csTimer backup happen on both google drive and "export to file" would make it much easier for future recovery.

**Comment by afedotov** [comment]
Actually ability to set autoexport frequency is already done, but option is currently hidden under DEBUG flag. Maybe just expose this to the users? And slightly extend option list to be more granular.

https://github.com/cs0x7f/cstimer/blob/45d1c6398cb49a2be98cec3b89450b851d6e8434/src/js/export.js#L592

**Comment by cs0x7f** [comment]
I am not planning to open this option at the moment. I think the current setting of 100 solves should be a relatively safe value. If it were changed to be smaller (or users were allowed to change it), the pressure on the server would become greater.

**Comment by Val-kaden** [comment]
@cs0x7f you're right but restricting that option to only the user's storage would no increase the stress on the servers right?
